src/Sum/README.md

```diff
@@ -0,0 +1,147 @@
+# Sum
+```haskell
+Sum Number
+```
+`Sum` is a `Monoid` that will combine two numbers using Addition.
+
+```js
```

Comment:
This example fails, it needs to wrap that 10 in a `Sum` before `concat`. Should read like:

```javascript
const Sum = require('crocks/Sum')
const mconcat = require('crocks/helpers/mconcat')
const mconcatMap = require('crocks/helpers/mconcatMap')

Sum(13)
//=> Sum 13

mconcat(Sum, [ 5, 2 ])
//=> Sum 7

Sum(10)
  .concat(Sum(10))
//=> Sum 20

// mapToSumByTen :: [ Number ] -> Sum
const sumByTen =
  mconcatMap(Sum, x => x * 10)

sumByTen([ 2, 2 ])
//=> Sum 40
```

otherwise this example as written throws.